rewrite-xml/src/main/java/org/openrewrite/xml/internal/XmlNamespaceUtils.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright 2024 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.xml.internal;
+
+import lombok.Value;
+import org.openrewrite.Cursor;
+import org.openrewrite.internal.lang.Nullable;
+import org.openrewrite.xml.tree.Xml;
+
+import java.util.Collection;
+import java.util.Map;
+
+public class XmlNamespaceUtils {
```

Comment:
Instead of a separate utils class, could this be methods on Namespaces instead you think?